database/src/episode.rs

```diff
@@ -311,99 +154,30 @@ impl Episode {
     /// * `season_num` - The season we are targetting
     /// * `ep_num` - Episode we are targetting
     ///
-    /// # Example
-    /// ```
-    /// use database::get_conn_devel as get_conn;
-    /// use database::library::{Library, InsertableLibrary, MediaType};
-    /// use database::media::{InsertableMedia, Media};
-    /// use database::tv::InsertableTVShow;
-    /// use database::season::InsertableSeason;
-    /// use database::episode::{InsertableEpisode, Episode};
-    ///
-    /// let conn = get_conn().unwrap();
-    ///
-    /// let library = InsertableLibrary {
-    ///     name: "test".into(),
-    ///     location: "/dev/null".to_string(),
-    ///     media_type: MediaType::Tv,
-    /// };
-    ///
-    /// let library_id = library.insert(&conn).unwrap();
-    ///
-    /// let new_show = InsertableMedia {
-    ///     library_id: library_id,
-    ///     name: "test".into(),
-    ///     added: "test".into(),
-    ///     media_type: MediaType::Tv,
-    ///     ..Default::default()
-    /// };
-    ///
-    /// let show_id = new_show.into_static::<InsertableTVShow>(&conn).unwrap();
-    ///
-    /// let new_season = InsertableSeason {
-    ///     season_number: 1,
-    ///     added: "test".into(),
-    ///     poster: "test".into(),
-    /// };
-    ///
-    /// let season_id = new_season.insert(&conn, show_id).unwrap();
-    ///
-    /// let new_episode = InsertableEpisode {
-    ///     media: InsertableMedia {
-    ///         library_id: library_id,
-    ///         name: "test_episode".into(),
-    ///         added: "test".into(),
-    ///         media_type: MediaType::Episode,
-    ///         ..Default::default()
-    ///     },
-    ///     seasonid: season_id,
-    ///     episode: 1,
-    /// };
-    ///
-    /// let episode_id = new_episode.insert(&conn, show_id).unwrap();
-    ///
-    /// // Get episode with show.id, season 1 and episode 1
-    /// let episode = Episode::get(&conn, show_id, 1, 1).unwrap();
-    ///
-    /// assert_eq!(episode.id, episode_id);
-    /// assert_eq!(episode.episode, 1);
-    ///
-    /// Library::delete(&conn, library_id).unwrap();
     pub async fn get(
```

Comment:
entire function can be collapsed into one query with 2 joins

Reply:
deferred